websites/L/LTTStore.com/presence.ts

```diff
@@ -0,0 +1,82 @@
+const presence = new Presence({
+	clientId: "1258619078201839646",
+});
+
+const enum Assets { // Other default assets can be found at index.d.ts
+	Logo = "https://www.lttstore.com/cdn/shop/files/LTT_Logo.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
+		buttons: [
+			{
+				label: "Visit LTTStore",
+				url: "https://www.lttstore.com/",
+			},
+		],
+	};
+
+	if (
+		document.location.hostname === "www.lttstore.com" ||
+		document.location.hostname === "lttstore.com"
+	) {
```

Comment:
No need for this, due to the url in metadata no other hostnames will use the presence.